### DISCUSSION TITLE
Adding more metrics for association rules

### DISCUSSION THREAD

**Discussion by Nachiket18** [rasbt/mlxtend#1031] [Ideas]
There are several metrics like lift, leverage, conviction etc are available for association rules. Wondering if it would be of help if we implement other metrics like Jaccard, Added Value, Certainty Factor etc.

**Comment by rasbt** [comment]
Thanks for bringing that up. Personally, I currently don't have the bandwidth to implement that, but I would welcome contributions and can help reviewing and adding them!

**Comment by RonakGSahu** [comment]
I'll be happy to lead this open source contribution with @Nachiket18 .

**Comment by ranykamel** [comment]
I will also be working on this contribution.

**Comment by Nachiket18** [comment]
Okay so the formulas that are necessary for implementing the new measures is provided here - https://mhahsler.github.io/arules/docs/measures

**Comment by RonakGSahu** [comment]
@rasbt We are starting to work on this contribution and were wondering if you could point us to where and how to add test cases as we already found out where to add the functionality in the files. Just asked this because some open-source library maintainers try to keep the codes in a certain fashion.

**Reply by rasbt** [reply]
Thanks for asking @RonakGSahu . I'd say it'd be best to add it here https://github.com/rasbt/mlxtend/tree/master/mlxtend/evaluate/tests (similar to test_lift_score.py)

**Comment by ranykamel** [comment]
@rasbt 
Hi, apologies for the disengagement, we are interested in revisiting this idea for the summer. @Nachiket18 and I were wondering if there are any library documentation and resources for the structure of the repository? What would be a good starting point?
We are also bringing on another contributor ([ankithn30](https://github.com/ankithn30)) onto this idea.
